src/core/load-fragment.c

```diff
@@ -427,38 +428,6 @@ int config_parse_socket_listen(const char *unit,
         return 0;
 }
 
-int config_parse_socket_bind(const char *unit,
-                             const char *filename,
-                             unsigned line,
-                             const char *section,
-                             unsigned section_line,
-                             const char *lvalue,
-                             int ltype,
-                             const char *rvalue,
-                             void *data,
-                             void *userdata) {
-
-        Socket *s;
-        SocketAddressBindIPv6Only b;
-
-        assert(filename);
-        assert(lvalue);
-        assert(rvalue);
-        assert(data);
-
-        s = SOCKET(data);
-
-        b = parse_socket_address_bind_ipv6_only_or_bool(rvalue);
```

Comment:
hmm, parse_socket_address_bind_ipv6_only_or_bool() should probably be renamed, to socket_address_bind_ipv6_or_bool_from_string() or so, since it returns -1 on failure, and not a negative errno...
And that means DEFINE_CONFIG_PARSE_ENUM()  would then be the right macro, and not 
DEFINE_CONFIG_PARSE()...

So much bugs in the old code regarding the -1 vs. -errno confusion!